package.json

```
@@ -146,6 +146,7 @@
     "publish-preminor": "node scripts/check-publish-access && node scripts/clear-package-dir preminor --verbose && lerna publish preminor --pre-dist-tag=next --preid=next --force-publish --allow-branch=master --message=\"chore(release): Publish next pre-minor\"",
     "publish-rc": "node scripts/check-publish-access && node scripts/clear-package-dir prerelease --verbose && lerna publish prerelease --pre-dist-tag=rc --preid=rc --message=\"chore(release): Publish rc\"",
     "publish-release": "node scripts/check-publish-access && node scripts/clear-package-dir patch --verbose && lerna publish patch",
+    "publish-v4": "node scripts/release-cli.js publish v4",
```

```suggestion
    "publish-v4": "node scripts/release-cli.js publish",
```
No longer needed after https://github.com/gatsbyjs/gatsby/pull/32694/commits/ab545b322e198fbd3faa78a170b9d8697e272c81